app/models/video.rb

```diff
@@ -12,25 +12,4 @@ def self.search_by_title(search_term)
     Video.where('lower(title) LIKE ?', "%#{search_term}%").order('created_at DESC')
```

Comment:
As long as you are using postgres on all of your servers, `ILIKE` is fine.  There are many parts of postgres that as not compatible with other database engines. I definitely see your point though :-)